src/components/QuestionPage.js

```diff
@@ -299,20 +348,19 @@ class QuestionPage extends Component {
         };
       }
     } else {
-     return{
-      height: 30,
-      width: 100,
-      justifyContent: 'center',
-      alignItems: 'center',
-      backgroundColor: '#a6a9b6',
-      borderRadius: 5,
-     } 
+      return {
+        height: 30,
+        width: 100,
+        justifyContent: 'center',
+        alignItems: 'center',
+        backgroundColor: '#a6a9b6',
+        borderRadius: 5,
+      };
```

Comment:
Stylelari if conditionlarina koymayalim. 3.0 a bunun ile ilgili yorum yapmistim.

Reply:
Bulamadım ama galiba düzelttiğim için bulamıyorum